src/cardContent.ts

```diff
@@ -25,7 +25,14 @@ export class SunCardContent {
         <div class="sun-card-text-container">
           <span class="sun-card-text-subtitle">${localization.Sunrise}</span>
           ${data?.times.sunrise ? this.generateTime(data.times.sunrise) : ''}
-
+        </div>
+        <div class="sun-card-text-container">
```

Comment:
Might be worth adding these as optionals, like with azimuth/elevation (and then probably in the same place as those)?

I had a go at this (before I realized you had started...), for inspiration: https://github.com/bjornomy/home-assistant-sun-card/tree/20-extra-time-information

Reply:
I think so, on one hand this is info is part of the original design but I forgot to include it, so I'm ok with it not being optional for now since there's an issue already for that (https://github.com/AitorDB/home-assistant-sun-card/issues/22). On the other hand I would like to not force people using the card to display more information that they may not want without the option to hide it.